api/src/Microsoft.Azure.IIoT.Api/src/Publisher/Models/EngineConfigurationApiModel.cs

```diff
@@ -42,5 +42,11 @@ public class EngineConfigurationApiModel {
             EmitDefaultValue = false)]
         public int? MaxMessageSize { get; set; }
 
+        /// <summary>
+        /// Max IoT D2C messages
```

Comment:
Maybe `Max IoT D2C outgress message buffer size` ? I would like to see here that this is buffer size, otherwise as a newcomer this summary and name of the parameter would be confusing for me.

Reply:
updated XML comment